lib/tasks/rake_config.rb

```diff
@@ -6,16 +6,30 @@ def context
 
     attr_writer :context
 
-    def config
+    def config(is_cc_local_worker: false)
```

Comment:
This function is used in many places in cc_ng.. 
if you want to access the config key is_cc_local_worker you could use RakeConfig.config.get(:key)